apm-agent-core/src/test/java/co/elastic/apm/agent/impl/ElasticApmTracerTest.java

```diff
@@ -489,4 +490,17 @@ void testOverrideServiceNameExplicitServiceName() {
         tracer.startTransaction(TraceContext.asRoot(), null, getClass().getClassLoader()).end();
         assertThat(reporter.getFirstTransaction().getTraceContext().getServiceName()).isNull();
     }
+
+    @Test
+    void testCaptureExceptionAndGetErrorId() {
+        Transaction transaction = tracerImpl.startTransaction(TraceContext.asRoot(), null, getClass().getClassLoader());
+        String errorId = tracerImpl.captureExceptionAndGetErrorId(System.currentTimeMillis() * 1000, new Exception("test"), transaction);
```

Comment:
Better call `transaction.captureException`, then you test both added methods.

Reply:
fixed